pkg/occlient/occlient_test.go

```diff
@@ -1611,52 +1611,6 @@ func TestUpdateBuildConfig(t *testing.T) {
 		updatedBuildConfig  buildv1.BuildConfig
 		wantErr             bool
 	}{
-		{
-			name:            "git to local with proper parameters",
```

Comment:
Its possible that I am missing something but, Why this test was removed?

Reply:
We no longer use `BuildConfig` which this test was testing for. That's why it was removed.

Reply:
Yes, the use-case is still valid but a different test should be written instead that doesn't check BuildConfig. The current test uses: `		updatedBuildConfig  buildv1.BuildConfig
` as an argument and that's no longer valid with updating from Git to Local